traveler-ios-ui/TravelerKitUI/ViewControllers/AvailabilityCalendarViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  CalendarViewController.swift
+//  TravelerKitUI
+//
+//  Created by Ben Ruan on 2019-10-24.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+protocol AvailabilityCalendarViewControllerDelegate: class {
+    func availabilityCalendarViewController(_ controller: AvailabilityCalendarViewController, didSelect availability: Availability)
+}
+
+class AvailabilityCalendarViewController: UIViewController {
+    var representingDate: Date?
+    var product: Product?
+    var availabilities: [Availability]?
+    var delegate: AvailabilityCalendarViewControllerDelegate?
+
+    override open func viewDidLoad() {
+        super.viewDidLoad()
+
+        if let _ = availabilities {
+            performSegue(withIdentifier: "resultSegue", sender: nil)
```

Comment:
I dont understand this logic.

Either the VC is given the availabilites, or it fetches them..not both